roles/docker/tasks/main.yml

```diff
@@ -1,10 +1,88 @@
 ---
 # tasks file for docker
+
+- name: Set node facts applicable to docker
```

Comment:
I think this might be contentious: if we ever want to e.g. configure Docker on a master, and not only on nodes, this might conflict and require rework.

Thing is, it seems right to do all the Docker config in one place, but there is at least portal_net that is node-specific.

If you see this as a problem, then the only alternative I see is to create a docker_node role.  Keeping the docker config in the openshift_node simply doesn't fly due to the fact you can't guarantee docker restart (in the handler) before node start (in the tasks), and you can't guarantee node SDN setup will have completed before docker restart.